pkgs/top-level/aliases.nix

```diff
@@ -320,6 +320,23 @@ mapAliases (rec {
   libva-full = libva;
   libva1-full = libva1;
 
+  # cross aliases
+
+  arm-frc-linux-gnueabi-eglibc = pkgsCross.arm-frc-linux-gnueabi.eglibc;
+  arm-frc-linux-gnueabi-linux-api-headers = pkgsCross.arm-frc-linux-gnueabi.buildPackages.linuxHeaders;
+  arm-frc-linux-gnueabi-binutils = pkgsCross.arm-frc-linux-gnueabi.buildPackages.binutils;
+  arm-frc-linux-gnueabi-gcc = pkgsCross.arm-frc-linux-gnueabi.buildPackages.gcc;
+
+  gcc-arm-embedded-4_8 = pkgsCross.armv7l-hf-multiplatform.buildPackages.gcc48;
```

Comment:
These don't look at all the same. `gcc-arm-embedded` has things like soft-float libraries, some embedded libc, etc.

Reply:
Ok I'll look into it. Is there another example platform that fits better? I can add a new one, my thought was that gcc-arm-embedded was just kind of a generic "arm" toolchain - I probably misunderstood the use case though.